graphql_compiler/tests/test_compiler.py

```diff
@@ -1719,8 +1718,26 @@ def test_fold_and_traverse(self):
                 $Animal___1___in_Animal_ParentOf =
                     Animal___1.in("Animal_ParentOf").out("Animal_ParentOf").asList()
         '''
+        expected_gremlin = '''
+            g.V('@class', 'Animal')
+            .as('Animal___1')
+            .transform{it, m -> new com.orientechnologies.orient.core.record.impl.ODocument([
+                animal_name: m.Animal___1.name,
+                sibling_and_self_names_list: (
+                    (m.Animal___1.in_Animal_ParentOf == null) ? [] : (
+                        m.Animal___1.in_Animal_ParentOf
+                            .collect{
+                                entry -> entry.outV.next()
+                            }
+                            .collectMany{entry -> entry.out_Animal_ParentOf.collect{
```

Comment:
Be consistent -- either the closure goes on its own line, or doesn't. Here you have two nested scopes but the line continuation on 1733 is only indented once.